group_list_users_test.go

```diff
@@ -16,13 +16,13 @@ func TestNewGroupListUsersE2E(t *testing.T) {
 	AssertEqual(t, users.Code, "Success")
 	AssertHasLength(t, users.Data.Items, 1)
 	AssertEqual(t, users.Data.Items[0].ID, PredefinedUserId)
-	AssertEqual(t, users.Data.Items[0].Email, "testingfivetran@gmail.com")
-	AssertEqual(t, users.Data.Items[0].GivenName, "Andrey")
-	AssertEqual(t, users.Data.Items[0].FamilyName, "Markov")
+	AssertEqual(t, users.Data.Items[0].Email, PredefinedUserEmail)
+	AssertEqual(t, users.Data.Items[0].GivenName, PredefinedUserGivenName)
+	AssertEqual(t, users.Data.Items[0].FamilyName, PredefinedUserFamilyName)
 	AssertEqual(t, *users.Data.Items[0].Verified, true)
 	AssertEqual(t, *users.Data.Items[0].Invited, false)
 	AssertEmpty(t, users.Data.Items[0].Picture)
-	AssertEqual(t, users.Data.Items[0].Phone, "89534322340")
+	AssertEqual(t, users.Data.Items[0].Phone, PredefinedUserPhone)
```

Comment:
This test will be rewritten further to avoid testing of pre-created user here.
Test should create a new group/user, then test get endpoints and then cleanup.